spec/integration/about_page_spec.rb

```diff
@@ -2,10 +2,15 @@
 
 describe 'about_page', :type => :request do
   before :each do
-    @current_user = double(:webauth_user, :login => 'sunetid', :logged_in? => true, :privgroup => ADMIN_GROUPS.first)
+    @current_user = double(
+      :webauth_user,
+      :login => 'sunetid',
+      :logged_in? => true,
+      :privgroup => User::ADMIN_GROUPS.first
+    )
     allow(@current_user).to receive(:is_admin).and_return(true)
-    allow(@current_user).to receive(:roles).and_return([])
     allow(@current_user).to receive(:is_manager).and_return(false)
+    allow(@current_user).to receive(:roles).and_return([])
```

Comment:
i also found this a bit unnecessary, but am :ok_hand: with it, in that i wouldn't stop the PR over it.